Form/EventListener/ResizePolyFormListener.php

```diff
@@ -84,7 +85,12 @@ public function __construct(array $prototypes, array $options = array(), $allowA
                 $defaultType = $type;
             }
 
-            $typeKey = $type instanceof FormTypeInterface ? $type->getName() : $type;
+            $typeKey = $type;
+
+            if ($type instanceof FormTypeInterface) {
```

Comment:
Builder::add and FormFactory::create* still accept a FormType without triggering a notice:

https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Form/FormBuilder.php#L78
https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Form/FormFactory.php#L68